libPhoneNumberTests/NBPhoneNumberUtilTest.m

```diff
@@ -1947,6 +1948,26 @@ - (void)testParseNationalNumberArgentina {
   XCTAssertTrue([arNumber isEqual:[_aUtil parse:@"023 1234 0000" defaultRegion:@"AR" error:&anError]]);
 }
 
+- (void)testParseNationalNumberRussia {
```

Comment:
How does this test validateNumberLength?

Reply:
It tests what was wrong with parse method and if that works fine now.